projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -30,6 +30,20 @@ class Jetpack_Memberships {
 	 */
 	public static $post_type_plan = 'jp_mem_plan';
 
+	/**
+	 * Tier type for plans
+	 *
+	 * @var string
+	 */
+	public static $tier_type = 'tier';
+
+	/**
+	 * Tier type for plans
+	 *
+	 * @var string
+	 */
+	public static $tier_type = 'tier';
```

Comment:
This seems to have been added by mistake.

Reply:
Nice cath! Fixed